pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -26,9 +26,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
-  nativeBuildInputs = [ glib wrapGAppsHook ocamlPackages.ocaml ]
-    ++ lib.optional enableX11 copyDesktopItems;
-  buildInputs = [ gsettings-desktop-schemas ncurses zlib ]
+  nativeBuildInputs = [ ocamlPackages.ocaml ]
+    ++ lib.optional enableX11 [ copyDesktopItems glib wrapGAppsHook ];
```

Comment:
I realized that `glib` can also go under `enableX11`, although there doesn't seem to be any additional reduction in closure size.

Reply:
Ah, of course. It's in `nativeBuildInputs`.